wamp-client/test.hs

```diff
@@ -60,7 +62,11 @@ testApp session = do
 
   putStrLn $ "\x2713 Session established: " ++ show (sessionId session)
 
-  res <- subscribe session "news" (Options dict) newsH >>= readMVar
+  -- res <- subscribe session "news" (Options dict) newsH >>= readMVar
+  res <- subscribe session "ticker" (Options dict) newsH >>= readMVar
+
+  threadDelay 5000000
```

Comment:
This looks suspicious. Why do we need a delay here?

Reply:
I don't think this was my change, I've inherited it from @cmahon 's changes I think

Reply:
I can't recall. Probably just a short-term experimental hack to overcome an issue I encountered. I only dabbled with the library briefly and just "got it running" so that I could test a broker API.